templates/docs/examples/templates/z-index.html

```diff
@@ -83,6 +83,11 @@
 <div class="p-stripe is-shallow">
   <button id="showModal" aria-controls="modal">Show modal&hellip;</button>
 
+  <button class="p-tooltip--top-left" aria-describedby="zindex-tooltip">
```

Comment:
Would be nice to use the detached tooltip and make it automatically visible, so this get's screenshoted by Percy.